torch/utils/_sympy/functions.py

```diff
@@ -1201,6 +1201,8 @@ def eval(cls, a):
                     a = sympy.oo
                 if a is -int_oo:
                     a = -sympy.oo
+                if name == "log2":
+                    return sympy.log(a, 2)
```

Comment:
Can we generalize this to any standard base?

Reply:
I guess... lol